src/SpiceSharpParser/ModelReaders/Netlist/Spice/Context/Models/PercentSpecification.cs

```diff
@@ -0,0 +1,11 @@
+﻿using SpiceSharpParser.Models.Netlist.Spice.Objects;
+
+namespace SpiceSharpParser.ModelReaders.Netlist.Spice.Context.Models
+{
+    public class PercentSpecification

```

Comment:
What does this class do? Is "Percent" relevant to its function, or is it used as a general description of a distribution specification (in which case the name is too narrow)?

Reply:
@marcin-golebiowski I've changed the name of the class to `ParameterRandomness`. It specifies what randomness to use for that parameter.